Cargo.toml

```diff
@@ -36,10 +36,11 @@ signatory = { version = "0.10", features = ["ed25519"] }
 signatory-dalek = "0.10"
 signatory-yubihsm = { version = "0.10", optional = true }
 subtle-encoding = "0.3"
-tendermint = { version = "0.2,0", path = "tendermint-rs" }
```

Comment:
I assume this was a typo? Guess it got parsed as a `VersionReq` of `0.2` or `0`. Weird.

Reply:
Ouch. Thanks for fixing this!